.github/dependabot.yml

```diff
@@ -0,0 +1,295 @@
+version: 2
+updates:
+  - package-ecosystem: "docker"
+    directory: "/nginx"
+    schedule:
+      interval: "weekly"
+    reviewers:
+      - "alismx"
+      - "DanielSass"
+      - "rin-skylight"
+      
+  - package-ecosystem: "docker"
+    directory: "/frontend"
+    schedule:
+      interval: "weekly"
+    reviewers:
+      - "alismx"
+      - "DanielSass"
+      - "rin-skylight"
+
+  - package-ecosystem: "docker"
+    directory: "/backend"
+    schedule:
+      interval: "weekly"
+    reviewers:
+      - "alismx"
+      - "DanielSass"
+      - "rin-skylight"
+
+  - package-ecosystem: "docker"
+    directory: "/backend/db-setup"
+    schedule:
+      interval: "weekly"
+    reviewers:
+      - "alismx"
+      - "DanielSass"
+      - "rin-skylight"
+
+  # - package-ecosystem: "github-actions"
+  #   directory: "/.github/workflows"
+  #   schedule:
+  #     interval: "weekly"
+  #   reviewers:
+      # - "alismx"
+      # - "emyl3"
+      # - "rin-skylight"
+
+  # - package-ecosystem: "github-actions"
```

Comment:
@alismx I didn't update the doc in time, but I'm also happy to be a reviewer for GH actions and Gradle!